integration-tests/opentelemetry-reactive/pom.xml

```diff
@@ -41,6 +41,12 @@
             <artifactId>opentelemetry-sdk-testing</artifactId>
         </dependency>
 
+        <!-- Security -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-elytron-security-properties-file</artifactId>
```

Comment:
Why do we need this now, @geoand ?

Reply:
got it. You are adding security to simulate early errors.

Reply:
Exactly - this was the easiest way to do that